core/src/main/java/com/linecorp/armeria/internal/common/util/TransportTypeProvider.java

```diff
@@ -196,9 +196,8 @@ private static TransportTypeProvider of(
             // TODO(trustin): Remove this block which works around the bug where loading both epoll and
             //                io_uring native libraries may revert the initialization of
             //                io.netty.channel.unix.Socket: https://github.com/netty/netty/issues/10909
-            final String unixSocketClassName = ChannelUtil.channelPackageName() + ".unix.Socket";
             try {
-                final Method initializeMethod = findClass(ChannelUtil.channelPackageName(), unixSocketClassName)
+                final Method initializeMethod = findClass(ChannelUtil.channelPackageName(), ".unix.Socket")
```

Comment:
nit; I don't think the method exists anymore in recent netty versions.
What do you think of modifying to use the correct method in recent versions?

https://github.com/netty/netty/blame/da1a448d5bc4f36cc1744db93fcaf64e198db2bd/transport-native-unix-common/src/main/java/io/netty/channel/unix/Socket.java#L556